cardano-api/src/Cardano/Api/TxSubmit/Types.hs

```diff
@@ -20,13 +20,13 @@ import qualified Data.Aeson as Aeson
 import           Data.Text (Text)
 import qualified Data.Text as Text
 
-import           Formatting ((%), build, sformat)
+import           Formatting (build, sformat, (%))
 
 
-data NodeApiEnv = NodeApiEnv
-  { naeConfig :: Genesis.Config
-  , naeSocket :: SocketPath
-  }
+data NodeApiEnv
+  = NodeApiEnv { naeConfig :: Genesis.Config
+               , naeSocket :: SocketPath
+               }
```

Comment:
Would be good to get something as close to what we had as possible.

Reply:
I've disabled record formatting altogether.